charts/kubernetes-external-secrets/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+name: kubernetes-external-secrets
+version: 0.1.0
```

Comment:
Should the `helm` chart version match the `appVersion`?

Reply:
Having a separate chart `version` from the `appVersion` lets us iterate and bump the chart content without changing the version of the app itself: https://github.com/helm/helm/blob/master/docs/charts.md#charts-and-versioning

Reply:
Thanks for pointing this out. Totally agree with it! Let' set `version: 1.0.0` then.